Objects/perf_trampoline.c

```diff
@@ -284,12 +284,23 @@ new_code_arena(void)
     void *start = &_Py_trampoline_func_start;
     void *end = &_Py_trampoline_func_end;
     size_t code_size = end - start;
+    // TODO: Check the effect of alignment of the code chunks. Initial investigation
+    // showed that this has no effect on performance in x86-64 or aarch64 and the current
+    // version has the advantage that the unwinder in GDB can unwind across JIT-ed code.
+    //
+    // We should check the values in the future and see if there is a
+    // measurable performance improvement by rounding trampolines up to 32-bit
+    // or 64-bit alignment.
 
     size_t n_copies = mem_size / code_size;
     for (size_t i = 0; i < n_copies; i++) {
         memcpy(memory + i * code_size, start, code_size * sizeof(char));
     }
     // Some systems may prevent us from creating executable code on the fly.
+    // TODO: Call icache invalidation intrinsics if available:
+    // __builtin___clear_cache/__clear_cache (depending if clang/gcc). This is
```

Comment:
The kernel would clear the icache for us. Not sure if this would ever be a problem.

Reply:
That's not correct in the general case. You are supposed to call this in aarch64 *if* you modify the code in some executable segment after it was read at least once. This doesn't apply for us but is not true that in all systems the kernel will clear it for us

Reply:
See the discussion in the previous pr